### ISSUE TITLE
Where do I find description to final raport ?

### ISSUE THREAD

**Issue by Adblu** [tensorpack/tensorpack#1440]
During training mask rcnn, following output is produced:

```
[0514 09:42:38 @base.py:283] Epoch 30 (global_step 1500) finished, time:2 minutes 8 seconds.
[0514 09:42:38 @graph.py:72] Running Op sync_variables/sync_variables_from_main_tower ...
[0514 09:42:39 @saver.py:82] Model saved to log/model-1500.
[0514 09:42:39 @misc.py:109] Estimated Time Left: 7 hours 45 minutes 47 seconds
[0514 09:42:39 @monitor.py:476] GPUUtil/0: 98.709
[0514 09:42:39 @monitor.py:476] GPUUtil/1: 98.85
[0514 09:42:39 @monitor.py:476] GPUUtil/2: 98.685
[0514 09:42:39 @monitor.py:476] GPUUtil/3: 98.661
[0514 09:42:39 @monitor.py:476] GPUUtil/4: 98.78
[0514 09:42:39 @monitor.py:476] GPUUtil/5: 98.472
[0514 09:42:39 @monitor.py:476] GPUUtil/6: 98.724
[0514 09:42:39 @monitor.py:476] GPUUtil/7: 98.559
[0514 09:42:39 @monitor.py:476] HostFreeMemory (GB): 462.02
[0514 09:42:39 @monitor.py:476] PeakMemory(MB)/gpu:0: 4892.9
[0514 09:42:39 @monitor.py:476] QueueInput/queue_size: 49.22
[0514 09:42:39 @monitor.py:476] Throughput (samples/sec): 3.1213
[0514 09:42:39 @monitor.py:476] fastrcnn_losses/box_loss: 0.22273
[0514 09:42:39 @monitor.py:476] fastrcnn_losses/label_loss: 0.18893
[0514 09:42:39 @monitor.py:476] fastrcnn_losses/label_metrics/accuracy: 0.92608
[0514 09:42:39 @monitor.py:476] fastrcnn_losses/label_metrics/false_negative: 0.64723
[0514 09:42:39 @monitor.py:476] fastrcnn_losses/label_metrics/fg_accuracy: 0.35277
[0514 09:42:39 @monitor.py:476] fastrcnn_losses/num_fg_label: 44.772
[0514 09:42:39 @monitor.py:476] learning_rate: 0.001
[0514 09:42:39 @monitor.py:476] maskrcnn_loss/accuracy: 0.81923
[0514 09:42:39 @monitor.py:476] maskrcnn_loss/fg_pixel_ratio: 0.4554
[0514 09:42:39 @monitor.py:476] maskrcnn_loss/maskrcnn_loss: 0.38417
[0514 09:42:39 @monitor.py:476] maskrcnn_loss/pos_accuracy: 0.36148
[0514 09:42:39 @monitor.py:476] mean_gt_box_area: 84321
[0514 09:42:39 @monitor.py:476] multilevel_roi_align/fpn_map_rois_to_levels/num_roi_level2: 262.1
[0514 09:42:39 @monitor.py:476] multilevel_roi_align/fpn_map_rois_to_levels/num_roi_level3: 130.92
[0514 09:42:39 @monitor.py:476] multilevel_roi_align/fpn_map_rois_to_levels/num_roi_level4: 94.363
[0514 09:42:39 @monitor.py:476] multilevel_roi_align/fpn_map_rois_to_levels/num_roi_level5: 24.621
[0514 09:42:39 @monitor.py:476] multilevel_roi_align_mask/fpn_map_rois_to_levels/num_roi_level2: 15.917
[0514 09:42:39 @monitor.py:476] multilevel_roi_align_mask/fpn_map_rois_to_levels/num_roi_level3: 13.202
[0514 09:42:39 @monitor.py:476] multilevel_roi_align_mask/fpn_map_rois_to_levels/num_roi_level4: 11.695
[0514 09:42:39 @monitor.py:476] multilevel_roi_align_mask/fpn_map_rois_to_levels/num_roi_level5: 3.9584
[0514 09:42:39 @monitor.py:476] rpn_losses/box_loss: 0.030642
[0514 09:42:39 @monitor.py:476] rpn_losses/label_loss: 0.046953
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/box_loss: 0.0056286
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/label_loss: 0.016631
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/label_metrics/precision_th0.1: 0.25786
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/label_metrics/precision_th0.2: 0.3269
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/label_metrics/precision_th0.5: 0.443
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/label_metrics/recall_th0.1: 0.57242
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/label_metrics/recall_th0.2: 0.56341
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/label_metrics/recall_th0.5: 0.53295
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/num_pos_anchor: 3.2213
[0514 09:42:39 @monitor.py:476] rpn_losses/level2/num_valid_anchor: 187.4
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/box_loss: 0.0050315
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/label_loss: 0.01069
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/label_metrics/precision_th0.1: 0.3816
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/label_metrics/precision_th0.2: 0.42788
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/label_metrics/precision_th0.5: 0.5732
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/label_metrics/recall_th0.1: 0.65188
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/label_metrics/recall_th0.2: 0.64589
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/label_metrics/recall_th0.5: 0.63354
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/num_pos_anchor: 5.1894
[0514 09:42:39 @monitor.py:476] rpn_losses/level3/num_valid_anchor: 45.586
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/box_loss: 0.0075027
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/label_loss: 0.014075
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/label_metrics/precision_th0.1: 0.60145
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/label_metrics/precision_th0.2: 0.60606
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/label_metrics/precision_th0.5: 0.63333
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/label_metrics/recall_th0.1: 0.7622
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/label_metrics/recall_th0.2: 0.70832
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/label_metrics/recall_th0.5: 0.64809
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/num_pos_anchor: 5.7753
[0514 09:42:39 @monitor.py:476] rpn_losses/level4/num_valid_anchor: 13.86
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/box_loss: 0.0068218
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/label_loss: 0.0042212
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/label_metrics/precision_th0.1: 0.70385
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/label_metrics/precision_th0.2: 0.72763
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/label_metrics/precision_th0.5: 0.76547
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/label_metrics/recall_th0.1: 0.81396
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/label_metrics/recall_th0.2: 0.81395
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/label_metrics/recall_th0.5: 0.80456
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/num_pos_anchor: 5.085
[0514 09:42:39 @monitor.py:476] rpn_losses/level5/num_valid_anchor: 6.4399
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/box_loss: 0.0056575
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/label_loss: 0.0013356
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/label_metrics/precision_th0.1: 0.61036
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/label_metrics/precision_th0.2: 0.61201
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/label_metrics/precision_th0.5: 0.63545
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/label_metrics/recall_th0.1: 0.67828
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/label_metrics/recall_th0.2: 0.67828
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/label_metrics/recall_th0.5: 0.67631
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/num_pos_anchor: 2.4978
[0514 09:42:39 @monitor.py:476] rpn_losses/level6/num_valid_anchor: 2.7118
[0514 09:42:39 @monitor.py:476] sample_fast_rcnn_targets/num_bg: 467.23
[0514 09:42:39 @monitor.py:476] sample_fast_rcnn_targets/num_fg: 44.772
[0514 09:42:39 @monitor.py:476] sample_fast_rcnn_targets/proposal_metrics/best_iou_per_gt: 0.74538
[0514 09:42:39 @monitor.py:476] sample_fast_rcnn_targets/proposal_metrics/recall_iou0.3: 0.99237
[0514 09:42:39 @monitor.py:476] sample_fast_rcnn_targets/proposal_metrics/recall_iou0.5: 0.97769
[0514 09:42:39 @monitor.py:476] total_cost: 1.1086
[0514 09:42:39 @monitor.py:476] wd_cost: 0.23514

```

Where do I find detailed description to those metrics ?

**Comment by ppwwyyxx** [comment]
There isn't a description of them now.
These are details inside the model that are probably relevant only to those very familiar with every part of the model.
